utils/log.py

```diff
@@ -557,9 +550,19 @@ def process(self, msg, kwargs):
 
 perflog = Perflog()
 
-# Capture warnings to the cfme logger using the warnings.showwarning hook
-warnings.showwarning = _showwarning
-warnings.simplefilter('default')
+
+def _configure_warnings():
```

Comment:
Something perhaps is wrong with your commit here, these shouldn't be part of this PR